README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
```

Comment:
Is there a (code) reason for the `postgresql_` prefix? Maybe just `version` or, to be more explicit, `server_version` would be better?